backend/app.js

```diff
@@ -54,8 +54,37 @@ app.post('/changeUsername', (req, res) => {
 })
 
 // TODO T32: log in 
-app.post('/login', (req, res) => {
-    res.status(200).send("Login endpoint under construction.");
+app.post('/login', async (req, res) => {
+    const { email, password } = req.body;
+
+    const storedHashedPassword = await prisma.user.findUnique({
+        where: {
+            email: email,
+        },
+        select: {
+            password: true,
+        },
+      });
```

Comment:
Add a try and catch block to gracefully catch database errors and return an error back to the client in the case where it can't find the email in the database.

Reply:
nit: variable name should be user because prisma will return a user object with only the password field.